CountItems/CountWorker.js

```diff
@@ -0,0 +1,144 @@
+const uuid = require('node-uuid');
+const { callOnce } = require('./utils');
+
+class CountWorker {
```

Comment:
Since this not actually worker code but the representation of a worker as seen from the master process (a proxy to a worker), I suggest a different name.